files/usr/share/cinnamon/applets/sound@cinnamon.org/applet.js

```diff
@@ -1420,6 +1420,11 @@ MyApplet.prototype = {
             this._control["change_" + type](device);
         });
 
+        let bin = new St.Bin({ x_align: St.Align.END, style_class: 'menuitem-detail' });
```

Comment:
Thanks for the feedback. I had a look at that style, but I think an inactive menu item should be able to be styled differently than context for an active menu item.